app/src/main/res/values/styles.xml

```diff
@@ -147,6 +149,8 @@
         <item name="toolbarDivider">@color/toolbar_divider_color_private_theme</item>
         <item name="fillLinkFromClipboard">@color/fill_link_from_clipboard_private_theme</item>
         <item name="menuCategoryText">@color/menu_category_private_theme</item>
+        <item name="syncDisconnected">@color/sync_disconnected_icon_fill_normal_theme</item>
```

Comment:
I think these two should be picking up the `sync_disconnected_icon_fill_private_theme` and `sync_disconnected_background_private_theme`?